crops-pipeline/Dockerfile

```diff
@@ -29,6 +29,10 @@ RUN /venv/bin/conda-unpack
 #The runtime base image.
 FROM ubuntu:${UBUNTU_VERSION}@sha256:${UBUNTU_DIGEST_SHA} AS runtime
 
+#Install necessary packages.
+RUN apt-get update -y && \
+    apt-get install wget -y
```

Comment:
There is a convention to clean up the layer at the end, like here

https://github.com/StatCan/daaas-containers/blob/2796464599253dd4415e9c9c6cfef9c4252386b3/btap/Dockerfile#L6

Otherwise the layer is huge and has unnecessary cached packages.